modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -41,22 +42,23 @@ namespace detail
     // types (aka Host type). Also it is widely used during serialization routine.
     enum class OpaqueKind: int
     {
-        CV_UNKNOWN,    // Unknown, generic, opaque-to-GAPI data type unsupported in graph seriallization
-        CV_BOOL,       // bool user G-API data
-        CV_INT,        // int user G-API data
-        CV_INT64,      // int64_t user G-API data
-        CV_DOUBLE,     // double user G-API data
-        CV_FLOAT,      // float user G-API data
-        CV_UINT64,     // uint64_t user G-API data
-        CV_STRING,     // std::string user G-API data
-        CV_POINT,      // cv::Point user G-API data
-        CV_POINT2F,    // cv::Point2f user G-API data
-        CV_POINT3F,    // cv::Point3f user G-API data
-        CV_SIZE,       // cv::Size user G-API data
-        CV_RECT,       // cv::Rect user G-API data
-        CV_SCALAR,     // cv::Scalar user G-API data
-        CV_MAT,        // cv::Mat user G-API data
-        CV_DRAW_PRIM,  // cv::gapi::wip::draw::Prim user G-API data
+        CV_UNKNOWN,            // Unknown, generic, opaque-to-GAPI data type unsupported in graph seriallization
+        CV_BOOL,               // bool user G-API data
+        CV_INT,                // int user G-API data
+        CV_INT64,              // int64_t user G-API data
+        CV_DOUBLE,             // double user G-API data
+        CV_FLOAT,              // float user G-API data
+        CV_UINT64,             // uint64_t user G-API data
+        CV_STRING,             // std::string user G-API data
+        CV_POINT,              // cv::Point user G-API data
+        CV_POINT2F,            // cv::Point2f user G-API data
+        CV_POINT3F,            // cv::Point3f user G-API data
+        CV_SIZE,               // cv::Size user G-API data
+        CV_RECT,               // cv::Rect user G-API data
+        CV_SCALAR,             // cv::Scalar user G-API data
+        CV_MAT,                // cv::Mat user G-API data
+        CV_DRAW_PRIM,          // cv::gapi::wip::draw::Prim user G-API data
+        CV_OT_TRACKING_STATUS  // cv::gapi::ot::TrackingStatus user G-API data
```

Comment:
I doubt if we need to know about this type at the Opaque level.

Reply:
Fixed!